docs/user-guide/trace/data-model.md

```diff
@@ -0,0 +1,198 @@
+---
+keywords: [Trace, OpenTelemetry, Jaeger]
+description: Covers internals of how trace data is stored in GreptimeDB.
+---
+
+# Trace Data Modeling
+
+:::warning
+
+This section currently in the experimental stage and may be adjusted in future versions.
+
+:::
+
+In this section, we will cover how trace data is modeled in GreptimeDB as
+tables.
+
+We reuse the concept of Pipeline for trace data modeling. However, note that at
```

Comment:
Consider linking to the pipeline documentation: https://docs.greptime.com/user-guide/logs/manage-pipelines.

Should we move pipeline management documentation out of the logs user guide and create a standalone document, similar to the flow documentation?

Reply:
Here I ignored link by purpose, because we only support non-customiziable built-in pipeline.